src/NetscriptWorker.ts

```diff
@@ -352,7 +352,9 @@ Otherwise, this can also occur if you have attempted to launch a script from a t
     .catch(function (e) {
       handleUnknownError(e, workerScript);
       killWorkerScript(workerScript);
-      workerScript.log("", () => (e instanceof ScriptDeath ? "Script killed." : "Script crashed due to an error."));
```

Comment:
I think we should leave this alone. As CatLover said, there are cases that will be missed if you take it out. Also, the timing with `sleep` and `asleep` is actually an important edge case that IMO we *should* be highlighting: `sleep` will "truly die" immediately, because it will get `delayReject`'ed. `asleep` may not, and if it does it could come much later. The only way of seeing this difference is the timing of the "Script killed" log message.

However, it might be appropriate to change the message, now that (afaik) all cases have an *additional* message at the time when killing is initiated. Perhaps "main() terminated."?